### DISCUSSION TITLE
HTML : Missing syntax highlighting in style and script blocks

### DISCUSSION THREAD

**Discussion by kresdjan** [emacs-tree-sitter/elisp-tree-sitter#184] [Q&A] [answered]
I am sure it is a configuration error on my part. But when I enable tree-sitter it breaks or removes syntax highlighting in `HTML+JS` and `HTML+CSS` modes.

Is this a tree-sitter misconfiguration or a conflict with the LSP server?

_with tree-sitter disabled_ ↓
<img width="1108" alt="without tree-sitter" src="https://user-images.githubusercontent.com/17638455/141674714-0c54a2b0-70e8-4d1d-981d-829fadd45aa3.png">

_with tree-sitter enabled_ ↓
<img width="1108" alt="with tree-sitter" src="https://user-images.githubusercontent.com/17638455/141674750-a547b276-bc2d-41ea-b47f-8a7183a2883a.png">

**Comment by ubolonton** [comment, marked as answer]
This is because the package `tree-sitter` doesn't currently handle embedded languages.

**Reply by elzorrorebelde** [reply]
see #33
